sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Program.cs

```diff
@@ -215,71 +174,40 @@ private static async Task RunOptions(Options opts)
-    /// <param name="roleConfiguration">The <see cref="RoleConfiguration" /> instance used to configure the role being run.</param>
-    /// <param name="metrics">The <see cref="Metrics" /> instance used by this role to send metrics to application insights.</param>
-    /// <param name="testName">The name of the test scenario that is being run, for metrics purposes.</param>
-    /// <param name="cancellationToken">The <see cref="CancellationToke"/> instance to signal the request to cancel the operation.</param>
+    /// <param name="testScenario">The <see cref="TestScenario"/> instance to convert.</param>
     ///
-    private static async Task RunRole(string role,
-                                      TestConfiguration testConfiguration,
-                                      RoleConfiguration roleConfiguration,
-                                      Metrics metrics,
-                                      string testName,
-                                      CancellationToken cancellationToken)
+    /// <returns>The string version of the <see cref="TestScenario"/> input.</returns>
+    ///
+    public static string TestScenarioToString(TestScenario testScenario) => testScenario switch
     {
-        if (role == RoleConfiguration.Publisher)
-        {
-            var publisherConfiguration = new PublisherConfiguration();
-            var publisher = new Publisher(publisherConfiguration, testConfiguration, metrics, testName);
-            await publisher.Start(cancellationToken);
-        }
-
-        if (role == RoleConfiguration.BufferedPublisher)
-        {
-            var publisherConfiguration = new BufferedPublisherConfiguration();
-            var publisher = new BufferedPublisher(testConfiguration, publisherConfiguration, metrics, testName);
-            await publisher.Start(cancellationToken);
-        }
-
-        if (role == RoleConfiguration.Processor)
-        {
-            var partitionCount = _getPartitionCount(testConfiguration.EventHubsConnectionString, testConfiguration.EventHub);
-            await partitionCount.ConfigureAwait(false);
-            var processConfiguration = new ProcessorConfiguration();
-            var processor = new Processor(testConfiguration, processConfiguration, metrics, partitionCount.Result, testName);
-            await processor.Start(cancellationToken);
-        }
-
-        if (role == RoleConfiguration.BurstBufferedPublisher)
-        {
-            var publisherConfiguration = new BufferedPublisherConfiguration();
-            publisherConfiguration.ProducerPublishingDelay = TimeSpan.FromMinutes(25);
-            var publisher = new BufferedPublisher(testConfiguration, publisherConfiguration, metrics, testName);
-            await publisher.Start(cancellationToken);
-        }
-
-        //return Task.CompletedTask;
-        await Task.Delay(TimeSpan.FromSeconds(1));
-    }
+        TestScenario.BufferedProducerTest => "BufferedProducerTest",
```

Comment:
Have we considered just using `ToString` here?  With the current mapping it seems like it would produce the same result.